### PR TITLE
New package: plex-media-player-2.19.1.904

### PR DESCRIPTION
**Pull request by Cogitri**
https://github.com/void-linux/void-packages/issues/722

This downloads plex' weblient stuff during buildtime as of now. It mostly consists of HTML&CSS stuff

### CODE REVIEW

File: srcpkgs/plex-media-player/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'plex-media-player'
+pkgname=plex-media-player
+version=2.18.0.893
+revision=1
+_commit_rev=48795f25
+wrksrc=plex-media-player-${version}-${_commit_rev}
+build_style=cmake
+configure_args="-DFULL_GIT_REVISION=${_commit_rev}"
+hostmakedepends="python3"
+makedepends="libcec libX11-devel libXext-devel libXrandr-devel MesaLib-devel
```
Comment (maxice8) [inline srcpkgs/plex-media-player/template]: libcec-devel?

**Comment by Cogitri** [comment]
Should build now

**Comment by Cogitri** [comment]
Seems like this doesn't cross compile due to `usr/$XBPS_CROSS_TRIPLET/lib/cmake/Qt5Core/Qt5CoreConfigExtrasMkspecDir.cmake` being faulty as far as I can see: It sets `set(_qt5_corelib_extra_includes "${_qt5Core_install_prefix}/../usr/lib/qt5//mkspecs/linux-g++")` instead of `set(_qt5_corelib_extra_includes "${_qt5Core_install_prefix}/../lib/qt5//mkspecs/linux-g++")` (no `/usr`).

File: srcpkgs/plex-media-player/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+short_desc="Next generation Plex Desktop/Embedded Client"
+maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://github.com/plexinc/plex-media-player"
+distfiles="https://github.com/plexinc/plex-media-player/archive/v${version}-${_commit_rev}.tar.gz"
+checksum=078e4baf8aa5688b021bcb5575cf365e252b1ca79c82737e5391fef01c38d3c9
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-devel qt5-host-tools"
+	configure_args+=" -DQTROOT=/usr/${XBPS_CROSS_TRIPLET}"
```
Comment (maxice8) [inline srcpkgs/plex-media-player/template]: why not just use ${XBPS_CROSS_BASE}/usr

if it isn't cross-compilation XBPS_CROSS_BASE will expand to nothing.

you can even move it to configure_args on top

Comment (Cogitri) [reply srcpkgs/plex-media-player/template]: Ah, thanks, that did the trick

**Comment by Cogitri** [comment]
Not quite sure if the libcec change is OK for RPI users (especially as I image that those don't want to compile libcec themselves), but otherwise we'd have to depend on rpi-userland and link applications that need libcec on armv{6,7} against some of rpi-userland's libs and I'm pretty sure that that would break runtime of those applications on anything but RPIs

**Pull request merged**